projectorganizer/src/prjorg-project.c

```diff
@@ -620,7 +620,7 @@ gint prjorg_project_add_properties_tab(GtkWidget *notebook)
 	gtk_entry_set_text(GTK_ENTRY(e->ignored_dirs_patterns), str);
 	g_free(str);
 
-	label = gtk_label_new(_("Generate tags for all project files:"));
+	label = gtk_label_new(_("Parse symbols for all project files:"));
```

Comment:
Yeah, and actually we don't want to "parse symbols" but rather files. I'd just simplify it to "Index all project files"